### ISSUE TITLE
The clustering accuracy is lower than the initial value

### ISSUE THREAD

**Issue by linguoting** [dawnranger/IDEC-pytorch#3]
Hello, after running idec.py, I found that the clustering accuracy obtained by Q is lower than the initial accuracy. During the training process, the accuracy first decreases and then increases, but finally it is lower than the accuracy of epoch0. I did not change the code and I used my own pre-trained model, what is the reason?
